scripts/agent-trader/agent_trader.py

```
@@ -281,6 +281,120 @@ def _mark_outcome(b):
+        f"${r['stake']:.0f}" if r["stake"] is not None else _MISSING,
+        str(r["marked_at"] or _MISSING), str(r["end_date"] or "")[:10],
+        (r["question"] or "")[:60],
+    ] for r in rows]
+    w = [max(len(h), *(len(row[i]) for row in body)) for i, h in enumerate(hdr)]
+    line = "  ".join(h.ljust(w[i]) for i, h in enumerate(hdr)).rstrip()
+    out = [line, "-" * len(line)]
+    out += ["  ".join(c.ljust(w[i]) for i, c in enumerate(row)).rstrip() for row in body]
+    out += [
+        "",
+        "(held) = frame of the token you own: YES token for a YES bet, NO token for a NO "
+        "bet.",
```

Ruff (0.16.1)

[warning] Unparenthesized implicit string concatenation in collection

Did you forget a comma?

(ISC004)